### PR TITLE
Milestone 5: Popup Window 

### PR DESCRIPTION
**Pull request by bushmusi**
# Milestone 5: Popup Window 

## Below features added

1. Popup page content added in `index.html` file
2. Add popup page style 
3. In our `app.js` file we have added project meta data and add flexible user interaction

Happy CR 👨‍💻 👩‍💻

### CODE REVIEW

File: js/app.js [hunk 1]
```diff
@@ -21,3 +25,68 @@ function addEvent(eventTagId) {
...
+  title: 'Multi-Post Stories',
+  text: "Multi-Post Stories is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and ",
+  mobileImage: './img/card2.svg',
+  tech: ['html', 'css', 'javaScript', 'Angular'],
+},
+{
+  title: 'Facebook 360',
+  text: "Facebook 360 is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and .",
+  mobileImage: './img/card3.svg',
+  tech: ['html', 'css', 'React'],
```
Comment (t-yanick) [inline js/app.js:49]: - [ ] As it is a requirement for this project, please make sure each object contains at least the following information about the project: `name`, `description`, `featuring image`, `technologies`, `link to live version`, and link to the `project source code`. Please the last two fields are missing, kindly add them as this make adding projects in the future easy, you will only add it in one place - as another object in the projects array.

File: js/app.js [hunk 2]
```diff
@@ -21,3 +25,68 @@ function addEvent(eventTagId) {
...
+  title: 'Tonic',
+  text: "Tonic is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and .",
+  mobileImage: './img/card1.png',
+  tech: ['html', 'css', 'javaScript'],
+},
+{
+  title: 'Multi-Post Stories',
+  text: "Multi-Post Stories is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and ",
+  mobileImage: './img/card2.svg',
+  tech: ['html', 'css', 'javaScript', 'Angular'],
```
Comment (t-yanick) [inline js/app.js:41]: - [ ] As it is a requirement for this project, please make sure each object contains at least the following information about the project: `name`, `description`, `featuring image`, `technologies`, `link to live version`, and link to the `project source code`. Please the last two fields are missing, kindly add them as this make adding projects in the future easy, you will only add it in one place - as another object in the projects array.

File: js/app.js [hunk 3]
```diff
@@ -21,3 +25,68 @@ function addEvent(eventTagId) {
 eventTagId.forEach(
   addEvent,
 );
+
+const projectContent = [{
+  title: 'Tonic',
+  text: "Tonic is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and .",
+  mobileImage: './img/card1.png',
+  tech: ['html', 'css', 'javaScript'],
```
Comment (t-yanick) [inline js/app.js:33]: - [ ] As it is a requirement for this project, please make sure each object contains at least the following information about the project: `name`, `description`, `featuring image`, `technologies`, `link to live version`, and link to the `project source code`. Please the last two fields are missing, kindly add them as this make adding projects in the future easy, you will only add it in one place - as another object in the projects array.

File: js/app.js [hunk 4]
```diff
@@ -21,3 +25,68 @@ function addEvent(eventTagId) {
...
+  title: 'Facebook 360',
+  text: "Facebook 360 is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and .",
+  mobileImage: './img/card3.svg',
+  tech: ['html', 'css', 'React'],
+},
+{
+  title: 'Uber Navigation',
+  text: "Uber Navigationis simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and .",
+  mobileImage: './img/card4.svg',
+  tech: ['html', 'css', 'RoR', 'html'],
```
Comment (t-yanick) [inline js/app.js:57]: - [ ] As it is a requirement for this project, please make sure each object contains at least the following information about the project: `name`, `description`, `featuring image`, `technologies`, `link to live version`, and link to the `project source code`. Please the last two fields are missing, kindly add them as this make adding projects in the future easy, you will only add it in one place - as another object in the projects array.

**Review by t-yanick — APPROVED** [review]
Hi @bushmusi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/app.js (hunk 4 above)
Comment (bushmusi) [reply js/app.js:57]: **Hi** @t-yanick 

You are right. It is mentioned in req.

_Thanks for reminding me. 🙇‍♂️_

**Review by ReshyResh — CHANGES_REQUESTED** [review]
# Hi team, 

Great work on making the change requested by a previous reviewer 👏 

You've done well implementing the requested changes, but it seems like the other reviewer missed an important requirement from the project guidelines, please check it below.

 ## Required Changes ♻️

- [ ] Please make sure that, as per requirement, the projects section in index.html isn't hard-coded, but is created dynamically from the Javascript objects. The html file should only have a container and the script should be responsible for adding each project to that container (Basically a for cycle where each object inside of the works array appends a div containing the work card to that container) .
- [ ] Please make sure the desktop popup is correctly centered. Currently, it has hard-coded pixel values which make it lean more to the left, kindly make sure to use margins correctly.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mateo951 — APPROVED** [review]
### Hi Team, 

Awesome job implementing all the changes required by your previous reviewers 🙌
Your project is complete! There is nothing else to say other than... it's time to merge it 😎 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**